interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -667,6 +679,9 @@ namespace cling {
                         IgnoreFilesFunc_t ignoreFiles =
                           [](const clang::PresumedLoc&) { return false;}) const;
 
+    void CodeComplete(const std::string& line, size_t& cursor,
```

Comment:
Yes, you are right it's evil. I will change the name. We do need it because codeComplete is an an internal function in the child interpreter whn it parses, whereas CodeComplete is the trigger of the parent interpreter to start the completion.